shell/edit/constraints.gatekeeper.sh.constraint/index.vue

```diff
@@ -356,7 +359,7 @@ export default {
             v-model:value="parametersYaml"
             class="yaml-editor"
             :editor-mode="editorMode"
-            @newObject="$set(value.spec, 'parameters', $event)"
+            @newObject="set(value.spec, 'parameters', $event)"
```

Comment:
This doesn't seem to set the `parameters` property correctly:


https://github.com/user-attachments/assets/f7d8c421-7ce3-4c3a-b286-672a2496ea98